### ISSUE TITLE
Improve the validator outputs

### ISSUE THREAD

**Issue by oeway** [bioimage-io/spec-bioimage-io#256]
Currently when we call the `validate` command, it returns `{}` if it passes the check, otherwise it produce for example the following:
```
{
  "{name: HPA Bestfitting Densenet, ...}": {
    "weights": {
      "onnx": {
        "value": [
          {
            "authors": [
              "Not a valid list."
            ],
            "weights_format": [
              "Must be equal to pytorch_state_dict."
            ]
          },
          {
            "authors": [
              "Not a valid list."
            ],
            "weights_format": [
              "Must be equal to pytorch_script."
            ]
          },
...
}
```

I am thinking it would be great if the output can be improved by the following:
 1. add a key, e.g. `success`, to explicitly say whether the validation is success
 2. Isn't the validate function used for validate for a single RDF? If that's the case I would change the top-level key (i.e. `"{name: HPA Bestfitting Densenet, ...}"` in the example) into `"error"`.

In other words, I propose the validate function output:
```json
{
    "success": true
}
```
or 
```json
{
    "success": false,
    "error": ...
}
```

@FynnBe Is there anything I missed here.

**Comment by FynnBe** [comment]
no, `nested_errors` is intended to hold errors of "nested" RDFs and only applies to collection RDFs.

I don't see how the `error` key could be missing from a `commands.validate` call:
https://github.com/bioimage-io/spec-bioimage-io/blob/8f5396652723a462d604a4f57ba5fdaefc7ce10d/bioimageio/spec/commands.py#L68-L74

Could you point me to where you call `validate` and make sure you are using the latest 0.3.3post4 ?

**Comment by oeway** [comment]
It should work now, @FynnBe in fact the function returned the right value. The issue here is that the website CI failed to deploy.
